.sdkmanrc

```diff
@@ -1,3 +1,3 @@
 # Enable auto-env through the sdkman_auto_env config
 # Add key=value pairs of SDKs to use below
-java=11.0.13-librca
+java=17.0.6-librca
```

Comment:
Had to update this with the orb update

Reply:
Makes sense, 17 has been the version bundled with Android Studio for a while now.